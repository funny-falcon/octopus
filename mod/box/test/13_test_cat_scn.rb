#!/usr/bin/ruby1.9.1

$:.push 'test/lib'
require 'standalone_env'

class Env < StandAloneEnv
  def config
    super + <<EOD
object_space[0].enabled = 1
object_space[0].index[0].type = "HASH"
object_space[0].index[0].unique = 1
object_space[0].index[0].key_field[0].fieldno = 0
object_space[0].index[0].key_field[0].type = "STR"
EOD
  end
end

env = Env.clean

env.with_server do
  16.times {|i| insert [i,i]}

  open('|./octopus --cat 10 2>/dev/null').lines do |l|
    puts l.gsub(/tm:\d+\.\d+ /, '')
  end
end
