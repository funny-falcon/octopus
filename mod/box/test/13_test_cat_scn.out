# box.insert([0, 0])
1

# box.insert([1, 1])
1

# box.insert([2, 2])
1

# box.insert([3, 3])
1

# box.insert([4, 4])
1

# box.insert([5, 5])
1

# box.insert([6, 6])
1

# box.insert([7, 7])
1

# box.insert([8, 8])
1

# box.insert([9, 9])
1

# box.insert([10, 10])
1

# box.insert([11, 11])
1

# box.insert([12, 12])
1

# box.insert([13, 13])
1

# box.insert([14, 14])
1

# box.insert([15, 15])
1

lsn:1 scn:1 t:1/snap_initial_tag 0.0.0.0:0 count:0 run_crc_log:0x00000000 run_crc_mod:0x00000000
lsn:1 scn:1 t:1/snap_final_tag 0.0.0.0:0 
lsn:2 scn:2 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <0:"\x00\x00\x00\x00", 0:"\x00\x00\x00\x00">
lsn:3 scn:3 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <1:"\x01\x00\x00\x00", 1:"\x01\x00\x00\x00">
lsn:4 scn:4 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <2:"\x02\x00\x00\x00", 2:"\x02\x00\x00\x00">
lsn:5 scn:5 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <3:"\x03\x00\x00\x00", 3:"\x03\x00\x00\x00">
lsn:6 scn:6 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <4:"\x04\x00\x00\x00", 4:"\x04\x00\x00\x00">
lsn:7 scn:7 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <5:"\x05\x00\x00\x00", 5:"\x05\x00\x00\x00">
lsn:8 scn:8 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <6:"\x06\x00\x00\x00", 6:"\x06\x00\x00\x00">
lsn:9 scn:9 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <7:"\x07\x00\x00\x00", 7:"\x07\x00\x00\x00">
lsn:10 scn:10 t:2/wal_tag 0.0.0.0:0 INSERT n:0 <8:"\x08\x00\x00\x00", 8:"\x08\x00\x00\x00">
